tests/conftest.py

```diff
@@ -13,22 +13,31 @@ def test_client():
 @pytest.fixture()
 def all_restaurants():
     return [{"address": "Minsk", "id": 0, "name": "Vasilki", "phone_number": "+375297777777",
-                             "work_time": "Monday-Sunday: 08:00 - 23:45"},
-                            {"address": "Minsk", "id": 1, "name": "Mama Doma", "phone_number": "+375336666666",
-                             "work_time": "Monday-Sunday: 10:00 - 22:00"}]
-
-
-@pytest.fixture()
-def mocker_restaurant_endpoint(mocker, all_restaurants):
-    def new_init(self, restaurants):
-        self.restaurants = all_restaurants
-    yield mocker.patch('delivery.app.RestaurantEndpoint.__init__', new_init)
-
-
-@pytest.fixture()
-def mocker_restaurant_item_endpoint(mocker, all_restaurants):
-    def new_init(self, restaurants):
-        self.restaurants = all_restaurants
-
-    yield mocker.patch('delivery.app.RestaurantItemEndpoint.__init__', new_init)
-
+             "work_time": "Monday-Sunday: 08:00 - 23:45"},
+            {"address": "Minsk", "id": 1, "name": "Mama Doma", "phone_number": "+375336666666",
+             "work_time": "Monday-Sunday: 10:00 - 22:00"}]
+
+
+@pytest.fixture(autouse=True)
+def mocker_get_all(mocker, all_restaurants):
+    def new_get_all(self):
+        return all_restaurants
+    yield mocker.patch('app.MemoryRestaurantRepository.get_all', new_get_all)
```

Comment:
1. it should be possible to use a single `patch` to mock the whole `app.MemoryRestaurantRepository`
2. now, when data access is performed in the repository, there should be no need to `patch` anything, it should be possible to use `MagicMock` instead, but I need to think how...

Reply:
I made mock of attribute restaurants = [] in Repository. I think that it is not bad.